contribute.md

```diff
@@ -267,14 +294,18 @@ ansible-playbook --skip-tags tests::cleanup \
 The [continuous integration](https://en.wikipedia.org/wiki/Continuous_integration) (CI)
 contains a set of automated tests that are triggered on a remote server. Some of them
 are immediately triggered when pushing new content to a PR (i.e. the tests hosted on
-TravisCI) while other need to be triggered by members of the project. This second
+Github Actions) while other need to be triggered by members of the project. This second
 set of tests can be manually triggered. To trigger them, write a command as a PR
 comment. The available commands are:
 
-- [citest] - Trigger a re-test for all machines.
-- [citest bad] - Trigger a re-test for all machines with an error or failure status.
-- [citest pending] - Trigger a re-test for all machines with a pending status.
-- [citest commit:<sha1\>] - specify a commit to be tested if the submitter is not trusted.
+- `[citest]` - Trigger a re-test for all machines.
+- `[citest bad]` - Trigger a re-test for all machines with an error or failure status.
+- `[citest pending]` - Trigger a re-test for all machines with a pending status.
+- `[citest commit:<sha1>]` - specify a commit to be tested if the submitter is not trusted.
+- `[citest skip]` - if you have a change to the documentation, or otherwise it
+would be a waste of time to do integration CI testing on your change, you can
+put `[citest skip]` in the title of your pull request.  This will save a lot of
+time.
```

Comment:
This may not be appropriate to put here, but once CI tests have started, there's no way to stop them. I.e., if you put `[citest]` (and its friends) repeatedly, they are all queued, aren't they? (I think I did it some time when the test was very slow and thought it hang, which was rather very harmful when the CI system is very busy and slow...) Maybe, `pending` is ok to put repeatedly, but not the others (except `skip`)?

Reply:
> This may not be appropriate to put here, but once CI tests have started, there's no way to stop them. I.e., if you put `[citest]` (and its friends) repeatedly, they are all queued, aren't they? (I think I did it some time when the test was very slow and thought it hang, which was rather very harmful when the CI system is very busy and slow...) Maybe, `pending` is ok to put repeatedly, but not the others (except `skip`)?

None of them should be used repeatedly - even `pending` will cause additional github traffic - `skip` can really only be used once, in the PR title